src/sidebar/util/dom.js

```diff
@@ -21,3 +21,25 @@ export function listen(element, events, listener, { useCapture = false } = {}) {
     );
   };
 }
+
+/**
+ * Obtain the pixel height of the element with id `elementId`, including
+ * top and bottom margins.
+ *
+ * @param {HTMLElement} element
+ * @return {number|null} - The element's height in pixels or `null` if no
+ *                         element with id `elementId` exists
+ */
+export function getElementHeightWithMargins(element) {
```

Comment:
Note this takes the HTML element rather than the `id` here, as IMO the knowledge of how to find the element to measure really belongs with the caller.